### PR TITLE
Fb jsp warnings 6160 daily b

### PR DESCRIPTION
**Pull request by ilikepies101**
This PR takes care of the warnings generated when the Daily B test suite is run that are not already covered in the BVT run (mostly strings / stringBuilders).

### CODE REVIEW

File: flow/src/org/labkey/flow/controllers/well/chooseGraph.jsp [hunk 1]
```diff
@@ -115,7 +116,7 @@
         <% if (hasScripts)
         {
         %>
-        <tr><td>Analysis Script:</td><td><select name="<%=FlowParam.scriptId%>" onchange="this.form.submit()">
+        <tr><td>Analysis Script:</td><td><select name="<%=h(FlowParam.scriptId)%>" onchange="this.form.submit()">
```
Comment (labkey-adam) [inline flow/src/org/labkey/flow/controllers/well/chooseGraph.jsp]: Should FlowParam implement EnumHasHtmlString? Then h() would be unnecessary.

File: flow/src/org/labkey/flow/controllers/well/showWell.jsp [hunk 2]
```diff
@@ -305,7 +305,7 @@ function showKeywords()
     for (var i=0 ; i<keywords.length ; i++)
     {
         var o = keywords[i];
-        o.label = o.index == 0 ? 'Keywords' : 'Parameter ' + _pad(o.index) + ' -- ' + o.name;
+        o.label = o.index == 0 ? h('Keywords') : h('Parameter ' + _pad(o.index) + ' -- ' + o.name);
```
Comment (labkey-adam) [inline flow/src/org/labkey/flow/controllers/well/showWell.jsp]: This is JavaScript code in the middle of a JSP! h() is not going to work there. I don't think you need any changes here.

File: flow/src/org/labkey/flow/controllers/well/showWell.jsp [hunk 3]
```diff
@@ -348,10 +348,10 @@ Ext.onReady(function()
     }
 });
 
-var statistics = <%=jsonStats%>;
+var statistics = <%=unsafe(jsonStats.toString())%>;
 var treeData;
 var stats;
-var keywords = <%=jsonKeywords%>;
+var keywords = <%=unsafe(jsonKeywords.toString())%>;
```
Comment (labkey-adam) [inline flow/src/org/labkey/flow/controllers/well/showWell.jsp]: No change needed here... but we really should make unsafe() and h() accept CharSequence instead of String. That would eliminate the need for these toString() calls. Soon...

File: ms2/src/org/labkey/ms2/proteinSequence.jsp [hunk 4]
```diff
@@ -38,4 +38,4 @@
         </tr>
     </table>
 <% } %>
-<big><tt><%= bean.protein.getFormattedSequence(bean.run) %></tt></big>
+<big><tt><%= h(bean.protein.getFormattedSequence(bean.run)) %></tt></big>
```
Comment (labkey-adam) [inline ms2/src/org/labkey/ms2/proteinSequence.jsp]: Rosaline had this change as well. Look at getFormattedSequence()... it generates HTML (see the tag constants). She already changed that method to return HtmlString, so you can revert this change.

File: flow/src/org/labkey/flow/controllers/well/chooseGraph.jsp (hunk 1 above)
Comment (ilikepies101) [reply flow/src/org/labkey/flow/controllers/well/chooseGraph.jsp]: It should!
